app/src/main/java/com/github/featuredetectandroid/utils/KeypointExtractionAlgorithm.kt

```diff
@@ -0,0 +1,17 @@
+package com.github.featuredetectandroid.utils
+
+enum class KeypointExtractionAlgorithm {
+    NONE,
+    SIFT,
+    SURF,
+    ORB,
+    SUPER_POINT
+}
+
+fun stringToAlgorithmMap() = mapOf(
```

Comment:
Try using `KeypointExtractionAlgorithm.valueOf()` for this

Reply:
I deleted `stringToAlgorithmMap()` for now.

Reply:
Now fixed.